docs/proposals/pepxxx_wheel_variant_support.md

```diff
@@ -1325,6 +1325,20 @@ Variant marker expressions MUST be evaluated against the variant properties stor
 against the current output of the provider plugins. If a non-variant wheel was selected or built, all variant markers
 evaluate to `False`.
 
+## How to Teach This
+
+For the transition period, during which some package managers do and some do not support variant wheels, users need to
+be aware that certain features may only be available with certain tools. Similarly, package maintainers should still
+publish non-variant wheels for a transition period.
+
+Users with specific deployment targets need to learn how to configure provider plugins for their target, for example,
+with a static file generated on the target.
+
+For authors of pure Python `py3-none-any` packages nothing changes. Authors of extension modules need to be taught how
```

Comment:
I would remove the first sentence. While indeed pure Python packages won't benefit from variants much, I don't think we have any technical limitations preventing them from coming up with a valid use case for them.